Feature : can switch in or out mock endpoint (#277)
unoplatform/uno.todo: README.md

# The Uno Platform ToDo reference App

> Staging builds are available here: https://happy-moss-09a619b10.1.azurestaticapps.net
> 
> Can trigger the mocked endpoint using the querystring ?mock=true

Reviewer: ```suggestion
- Staging builds are available here: https://happy-moss-09a619b10.1.azurestaticapps.net
- Staging builds with mocked endpoint are available here: https://happy-moss-09a619b10.1.azurestaticapps.net/?mock=true
```
